components/script/dom/readablestreamdefaultreader.rs

```diff
@@ -70,3 +82,13 @@ impl ReadableStreamDefaultReaderMethods for ReadableStreamDefaultReader {
         todo!()
     }
 }
+
+#[derive(Clone, JSTraceable, MallocSizeOf)]
+pub struct ReadRequest {
+    #[ignore_malloc_size_of = "Rc"]
+    chunk_steps: Rc<Function>,
```

Comment:
These steps are algorithms instead of JS Function. I guess let's just make an empty struct first.

Reply:
ok